src/explore-education-statistics-common/src/modules/table-tool/components/DataSetStep.module.scss

```diff
@@ -0,0 +1,7 @@
+@import '~govuk-frontend/govuk/base';
+
+.viewAll {
```

Comment:
Rather than create a custom style for this, it could be better to add a `labelClassName` prop to `FormRadio` so we can provide `govuk-!-font-weight-bold` to the `RHFFormFieldRadioGroup` radio option i.e.

```tsx

<RHFFormFieldRadioGroup
  options={
    renderFeaturedTable && featuredTables.length
      ? [
          {
            label: 'View all featured tables',
            value: 'all-featured',
            labelClassName: 'govuk-!-font-weight-bold',
            divider: 'or select a data set',
          },
          ...options,
        ]
      : options
  }
/>
```